<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>641</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rectanglePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rect</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="angleSpinBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>90</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>120</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scaleButton">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>60</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="widthLabel">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>149</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLabel" name="heightLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>149</y>
      <width>62</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLable">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>39</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scaleSpinBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="widthSpinBox">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setPushButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>270</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QLabel" name="angleLable">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>angle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSpinBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>170</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="trianglePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>triangle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pentagonPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pentagon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hexagonPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hexagon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="circlePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="areaPushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>380</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>get area and perimetr</string>
    </property>
   </widget>
   <widget class="QLabel" name="AreaAndPerimetrLabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>330</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="radiusSpinBox">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>170</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="radiusLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>radius</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="XspinBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>220</y>
      <width>43</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="YspinBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>220</y>
      <width>43</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xlabel">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="ylabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>200</y>
      <width>57</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>999</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
